torch/functional.py

```diff
@@ -326,6 +328,71 @@ def parse_subscript(n: int) -> str:
 
     return _VF.einsum(equation, operands)  # type: ignore[attr-defined]
 
+# Wrapper around _histogramdd and _histogramdd_bin_edges needed due to (Tensor, Tensor[]) return type.
+if TYPE_CHECKING:
+    # The JIT doesn't understand Union, so leave untyped for mypy
+    pass
+else:
+    def histogramdd(input: Tensor,
+                    bins: Union[List[Tensor], List[int]],
+                    range: Optional[List[float]] = None,
+                    weight: Optional[Tensor] = None,
+                    density: bool = False):
+        r"""
+        histogramdd(input, bins, *, range=None, weight=None, density=False, out=None) -> (Tensor, Tensor[])
+
+        Computes a histogram of the values in a tensor. Interprets the elements of an
+        input tensor with innermost dimension D as D-dimensional coordinates.
+
+        :attr:`bins` can be a sequence of D ints or a sequence of D 1D tensors.
```

Comment:
Before getting into bins, what is a multidimensional histogram? If the example idea above sounds good maybe it could be coupled with an additional sentence that explains what a user would expect to get back, and how tensors with 3+ dims and basically flattened into the matrix case?

Reply:
Like the big thing here is that the histogram is applied independently along each dimension, right?